velero/README.md

```diff
@@ -33,6 +35,42 @@ Therefore, make sure to update the annotations of the `velero` deployment as wel
 <!-- xxz tab xxx -->
 <!-- xxz tabs xxx -->
 
+#### Logs
+
+The Velero integration can collect logs from the Velero pods.
+
+<!-- xxx tabs xxx -->
+<!-- xxx tab "Host" xxx -->
+
+Apply this if you want to collect logs from Velero containers on a host.
+
+1. Collecting logs is disabled by default in the Datadog Agent. Enable it in your `datadog.yaml` file:
+
+   ```yaml
+   logs_enabled: true
+   ```
+
+2. Uncomment and edit the logs configuration block in your `velero.d/conf.yaml` file. Here's an example:
+
+   ```yaml
+   logs:
+     - type: docker
+       source: velero
+       service: velero
+   ```
+
+<!-- xxz tab xxx -->
+<!-- xxx tab "Kubernetes" xxx -->
+
+Apply this if you want to collect logs from a Velero Kubernetes deployment.
+
+Collecting logs is disabled by default in the Datadog Agent. To enable it, see [Kubernetes Log Collection][10].
+
+Then, set Log Integrations as pod annotations. This can also be configured with a file, a configmap, or a key-value store. For more information, see the configuration section of [Kubernetes Log Collection][11].
```

Comment:
Recommend following the same format as the Host tab.

```suggestion
1. Collecting logs is disabled by default in the Datadog Agent. To enable it, see [Kubernetes Log Collection][10].

2. Set Log Integrations as pod annotations. This can also be configured with a file, a ConfigMap, or a key-value store. For more information, see the configuration section of [Kubernetes Log Collection][11].
```